targets/openssl/patches/bugs/101.patch

```diff
@@ -0,0 +1,19 @@
+--- a/crypto/evp/evp_enc.c
++++ b/crypto/evp/evp_enc.c
+@@ -656,7 +656,16 @@ static int evp_EncryptDecryptUpdate(EVP_CIPHER_CTX *ctx,
+     i = ctx->buf_len;
+     OPENSSL_assert(bl <= (int)sizeof(ctx->buf));
+     if (i != 0) {
++
++#ifdef MAGMA_ENABLE_CANARIES
++        MAGMA_LOG(101,INT_MAX - inl < i);
```

Comment:
Move this bug to a "graveyard" folder instead of "bugs".
The original bug report states that no internal instance of the function call can trigger this bug.
Although it could possibly be triggered by someone calling the function directly, we're not interested in that right now.
So instead of deleting it, I suggest moving it to a graveyard directory in case it comes into use later.